onnxscript/ir/_core.py

```diff
@@ -943,35 +967,35 @@ def __str__(self) -> str:
         # that make them hard to read
         return f"%{_quoted(value_name)}<{type_text},{shape_text}>"
 
-    def def_node(self) -> Node | None:
+    def producer(self) -> Node | None:
         """The node that produces this value."""
-        return self._def_node
+        return self._producer
 
-    def def_index(self) -> int | None:
+    def index(self) -> int | None:
         """The index of the output of the defining node."""
-        return self._def_index
+        return self._index
 
-    def users(self) -> frozenset[tuple[Node, int]]:
-        """Return a set of users of the value.
+    def consumers(self) -> frozenset[tuple[Node, int]]:
+        """Return a set of consumers of the value.
 
         The set contains tuples of ``(Node, index)`` where the index is the index of the input
         of the node. For example, if ``node.inputs[1] == value``, then the user is ``(node, 1)``.
         """
-        return frozenset(self._users)
+        return frozenset(self._consumers)
 
     def _add_user(self, user: Node, index: int) -> None:
```

Comment:
should this be renamed to `_add_consumer` ?

Reply:
Done